testsuite/forge/src/backend/k8s/cluster_helper.rs

```diff
@@ -300,6 +302,12 @@ pub fn clean_k8s_cluster(
         .join("diem_status.json")
         .display()
         .to_string();
+
+    // run genesis from the directory in diem/init image
+    let mut move_modules_dir = GENESIS_MODULES_DIR.to_string();
```

Comment:
you can do this part as:
```
let move_modules_dir = if let Some(genesis_modules_path) = genesis_modules_path {
     genesis_modules_path
} else {
      GENESIS_MODULES_DIR.to_string()
}
```

and maybe rename `GENESIS_MODULES_DIR` to `DEFAULT_GENESIS_MODULES_DIR` if I understand correct